### ISSUE TITLE
Funds dropdown in Donation Form: UI is broken

### ISSUE THREAD

**Issue by Usama-Bin-Khalid-98** [impress-org/givewp#7167]
## User Story
If there are multiple funds, the funds drop down clips the last ones and is not properly scrollable.

## Visuals
<img width="1331" alt="Screenshot 2024-01-03 at 6 31 26 PM" src="https://github.com/impress-org/givewp/assets/121162049/4b83f2b8-2797-47df-8cc7-306b4c0b8cf8">

**Comment by JoshuaHungDinh** [comment]
Hi @Usama-Bin-Khalid-98 

Thank you for highlighting this concern. I've initiated an internal ticket to address and prioritize your feedback. Since it is specific to the Funds and Designations add-on and not our core plugin, I've closed the issue. I might suggest posting any future bug reports through this public channel https://feedback.givewp.com/bug-reports - unless you are including a PR. This will allow others to vote on your feedback, as well as allow you to follow the internal status of the issue.
